src/amuse/community/fi/interface.py

```diff
@@ -2608,1484 +2769,1433 @@ def define_methods(self, handler):
+            ),
         )
-        
+
+        handler.add_method("set_masscrit", (units.MSun,), (handler.ERROR_CODE,))
+
         handler.add_method(
             "get_sfeff",
             (),
-            (handler.NO_UNIT, handler.ERROR_CODE,)
-        )
-        
-        handler.add_method(
-            "set_sfeff",
-            (handler.NO_UNIT, ),
-            (handler.ERROR_CODE,)
+            (
+                handler.NO_UNIT,
+                handler.ERROR_CODE,
+            ),
         )
-        
+
+        handler.add_method("set_sfeff", (handler.NO_UNIT,), (handler.ERROR_CODE,))
+
         handler.add_method(
             "get_tbubble",
             (),
-            (units.Myr, handler.ERROR_CODE,)
-        )
-        
-        handler.add_method(
-            "set_tbubble",
-            (units.Myr, ),
-            (handler.ERROR_CODE,)
+            (
+                units.Myr,
+                handler.ERROR_CODE,
+            ),
         )
-        
+
+        handler.add_method("set_tbubble", (units.Myr,), (handler.ERROR_CODE,))
+
         handler.add_method(
             "get_sne_eff",
             (),
-            (handler.NO_UNIT, handler.ERROR_CODE,)
-        )
-        
-        handler.add_method(
-            "set_sne_eff",
-            (handler.NO_UNIT, ),
-            (handler.ERROR_CODE,)
+            (
+                handler.NO_UNIT,
+                handler.ERROR_CODE,
+            ),
         )
-        
+
+        handler.add_method("set_sne_eff", (handler.NO_UNIT,), (handler.ERROR_CODE,))
+
         handler.add_method(
             "get_tsnbeg",
             (),
-            (units.Myr, handler.ERROR_CODE,)
-        )
-        
-        handler.add_method(
-            "set_tsnbeg",
-            (units.Myr, ),
-            (handler.ERROR_CODE,)
+            (
+                units.Myr,
+                handler.ERROR_CODE,
+            ),
         )
-        
+
+        handler.add_method("set_tsnbeg", (units.Myr,), (handler.ERROR_CODE,))
+
         handler.add_method(
             "get_rhomax",
             (),
-            (nbody_system.density, handler.ERROR_CODE,)
-        )
-        
-        handler.add_method(
-            "set_rhomax",
-            (nbody_system.density, ),
-            (handler.ERROR_CODE,)
+            (
+                nbody_system.density,
+                handler.ERROR_CODE,
+            ),
         )
-        
+
+        handler.add_method("set_rhomax", (nbody_system.density,), (handler.ERROR_CODE,))
+
         handler.add_method(
             "get_halofile",
             (),
-            (handler.NO_UNIT, handler.ERROR_CODE,)
-        )
-        
-        handler.add_method(
-            "set_halofile",
-            (handler.NO_UNIT,),
-            (handler.ERROR_CODE,)
+            (
+                handler.NO_UNIT,
+                handler.ERROR_CODE,
+            ),
         )
-        
+
+        handler.add_method("set_halofile", (handler.NO_UNIT,), (handler.ERROR_CODE,))
```

Comment:
Now this is inconsistent with everything else... Again, ah well...

Reply:
Yes, not sure why.